config/sonar.dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM openjdk:8
```

Comment:
why do we need sonar docker file? How is it different from original from Sonar ?
We need to reuse existing sonar image that is hosted on dockerhub.